bots/pr/src/test/java/org/openjdk/skara/bots/pr/CSRTests.java

```diff
@@ -449,10 +449,10 @@ void notYetApprovedCSR(TestInfo testInfo) throws IOException {
             prAsReviewer.addComment("/csr");
             TestBotRunner.runPeriodicItems(prBot);
 
-            // The bot should reply with a message that the PR will not be integrated until the CSR is approved
-            assertLastCommentContains(pr, "this pull request will not be integrated until the [CSR]");
-            assertLastCommentContains(pr, "for issue ");
-            assertLastCommentContains(pr, "has been approved.");
+            // The bot should reply with a message that there is already an approved
+            // Now CheckWorkItem is responsible for updating CSR label, so before 'csr' is handled, csr label is added to this pr
```

Comment:
I don't understand this comment.

Reply:
Sorry, somehow the sentence is truncated.  Will update the comment